andela/ah-frontend-poseidon
# #162162966: Users should be able to view and edit their profiles

**What does this PR do?**

Creates UI functionality for users to view and edit the profiles.

**Description of the tasks to be completed?**
- Make the profile button functional. It should redirect user to profile page.
- Create profile page with edit functionality.

**How should this be manually tested?**
- Clone the repository
- git checkout ft-profile-view-edit-162162966
- Install dependencies using `yarn install`
- Run the app using `yarn start`
- The application should be running.
- Open browser and go to http://localhost:3000
- When logged in, click Profile and view/edit profile.

**What are the related pivotal tracker stories?**
#162162966

**Screenshots**

<img width="1431" alt="screen shot 2019-01-21 at 18 43 05" src="https://user-images.githubusercontent.com/40595048/51484426-91d8dd80-1dac-11e9-8e81-62007f2f7557.png">
<img width="1425" alt="screen shot 2019-01-21 at 18 43 20" src="https://user-images.githubusercontent.com/40595048/51484447-a2895380-1dac-11e9-80c1-7ea7ae2c22f4.png">
<img width="1192" alt="screen shot 2019-01-17 at 23 31 57" src="https://user-images.githubusercontent.com/40595048/51347225-8f2a6f80-1ab0-11e9-9beb-909928bc4bb0.png">
<img width="1102" alt="screen shot 2019-01-17 at 23 32 09" src="https://user-images.githubusercontent.com/40595048/51347239-99e50480-1ab0-11e9-8fc5-c9931a798302.png">
